lib/models/application-invite.ts

```diff
@@ -164,7 +165,7 @@ const getApplicationInviteModel = function (
 								$top: 1,
 								$select: ['id'],
 								$filter: {
-									name: roleName,
+									name: roleName as ApplicationMembershipRoles,
```

Comment:
Shouldn't we add this typing to `ApplicationInviteOptions` instead? I see it's destructured from there.